rules/windows/execution_register_server_program_connecting_to_the_internet.toml

```diff
@@ -28,7 +28,7 @@ type = "query"
 
 query = '''
 event.category:network and event.type:connection and
-  process.name:(regsvr32.exe or regsvr64.exe) and
+  process.name:(regsvr32.exe or regsvr64.exe or RegAsm.exe) and
```

Comment:
Since `RegAsm.exe` was added, we could also add that technique to the Mitre metadata https://attack.mitre.org/techniques/T1121/

I'd also ensure that winlogbeat correctly picks up the casing  of `RegAsm.exe` to ensure the rule fires as intended.

Reply:
added TID and program name is verified to match windows default one.